ui_src/src/components/tagList/style.scss

```diff
@@ -0,0 +1,77 @@
+.tags-list-wrapper {
+    display: flex;
+    align-items: center;
+    margin-left: 5px;
+
+    .plus-tags {
+        background-color: rgba(132, 129, 138, 0.1);
+        padding-right: 5px;
+        border-style: solid;
+        border-color: transparent;
```

Comment:
If the border-color is transparent, why did you set the border style to solid and the border-width to thin?
Don't set the border at all.